ew/utils/frontend.py

```diff
@@ -496,14 +496,31 @@ async def update_slimernalia_kingpin(client, server):
     # Update the new kingpin of slimernalia
     new_kingpin = EwUser(id_user=ewutils.get_most_festive(server), id_server=server.id)
     new_kingpin.slimernalia_kingpin = True
+    
+    # Reset the new kingpin's festivity upon getting the award
+    old_festivity = new_kingpin.festivity
```

Comment:
the current term is 6 hours, it triggers at the same time as leaderboards. are you sure we should be resetting it that quickly?

Reply:
we'll test the 6 hour period live on the test bot, if the community finds it too harsh or stifling we can always move to a 12 hour period